tests/Php82/SensitiveParameterValuePolyfillTest.php

```diff
@@ -17,7 +17,7 @@
 /**
  * @author Tim Düsterhus <duesterhus@woltlab.com>
  */
-class SensitiveParameterValueTest extends TestCase
+class SensitiveParameterValuePolyfillTest extends TestCase
```

Comment:
TestListenerTrait tries to find a polyfill declaration in `bootstrap.php` base on the test classname. However, `SensitiveParameterValue` is a "special" polyfill that doesn't need any declaration in `bootstrap.php`.

So, PHPUnit warned that no polyfill was found, but that's normal. By renaming, the file to not match the `SensitiveParametrerValue` classname, we avoid `TestListenerTrait` to raise a wrong warning.